examples/bigcommerce/README.md

```diff
@@ -203,6 +212,12 @@ const slug = ctx.params?.slug
 const product = await getProduct(Number.parseInt(slug.toString(), 10))
 ```
 
+### Why is the "dynamic product route" using [ISR](https://nextjs.org/docs/basic-features/data-fetching/incremental-static-regeneration) when the root catch all route is not?
```

Comment:
Is this the first time we use the word "root" to describe the catch all route? It threw me off since before it was always "optional catch all route"